### ISSUE TITLE
Graph Module for FMS

### ISSUE THREAD

**Issue by UmangBR** [amosproj/amos2022ss03-turtlebot-fleet-management#37]
## User story
1. As a SD
2. I want to develop a graph module
3. So that I can define a storage format and provide functions for traversal

## Acceptance criteria
* Be able to define a storage format for FMS Python graphs
* Be able to define functions for later requests for traversal. 

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by UmangBR** [comment]
Later requst:  „hey whats the shortest route from A to B“

**Comment by Venkatesh770** [comment]
![imported_vmap](https://user-images.githubusercontent.com/69533541/170219580-e1e88015-48e5-4fff-bb9a-522d9e5b71f3.png)

The Dijkstra Algorithm is followed to find the Shortest Distance Between the Nodes.
For a Default Graph from Vmap it was Difficult to Implement 
As a First Step or Skeleton Preparation , The Algorithm is implemented for above simple map and it works successfully
